libdebug/architectures/aarch64/aarch64_stack_unwinder.py

```diff
@@ -0,0 +1,68 @@
+#
+# This file is part of libdebug Python library (https://github.com/libdebug/libdebug).
+# Copyright (c) 2023-2024 Roberto Alessandro Bertolini, Francesco Panebianco, Gabriele Digregorio. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for details.
+#
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from libdebug.architectures.stack_unwinding_manager import StackUnwindingManager
+
+if TYPE_CHECKING:
+    from libdebug.state.thread_context import ThreadContext
+
+
+class Aarch64StackUnwinder(StackUnwindingManager):
+    """Class that provides stack unwinding for the AArch64 architecture."""
+
+    def unwind(self: Aarch64StackUnwinder, target: ThreadContext) -> list:
+        """Unwind the stack of a process.
+
+        Args:
+            target (ThreadContext): The target ThreadContext.
+
+        Returns:
+            list: A list of return addresses.
+        """
+        assert hasattr(target.regs, "pc")
```

Comment:
why this check? I believe we can do a better exception managment in this case

Reply:
That's not a real check, it is there so the linter can infer that the "pc" attribute exists on what it believes to be a generic register file.

Reply:
I see, but it’s a very ugly line of code added just for the linter. Are there really no better alternatives? (Using getattr doesn't seem much better to me)

Reply:
Probably `target.regs.pc: int` would work too, but I personally find it even uglier